README.md

```diff
@@ -21,7 +20,7 @@ queue.startPolling(
     callback();
   },
   function error(queueError) {
-    throw queueError;
+    console.error(queueError);
```

Comment:
Yeah, the idea here would be that sawmill logs the `queueError` however it wants